### PR TITLE
Add function to display information message

### PR DESCRIPTION
**Pull request by SebBareyre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | add a function to display information message in backoffice
| Type?         | new feature
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | -
| How to test?  | Just add for example `echo $this->displayInformation("Your message here");` in the function `getContent()` of a test module

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9344)

### CODE REVIEW

**Comment by PierreRambaud** [comment]
@eternoendless @Quetzacoalt91 Look legit to have the same system as `displayConfirmation` even if I hate html in php files :( wdty?

**Comment by Quetzacoalt91** [comment]
Well, the system exists since a very long time, can't blame @SebBareyre to reuse it.

However, why don't we allow an array like in the other methods?

**Comment by SebBareyre** [comment]
Why not @Quetzacoalt91 :) 
Do you want I close this PR and propose another one ?

**Comment by Quetzacoalt91** [comment]
You can update this one too. :)

**Review by SebBareyre — COMMENTED** [review]
Voilà :)

File: classes/module/Module.php [hunk 1]
```diff
@@ -2047,6 +2047,34 @@ public function displayConfirmation($string)
         </div>';
         return $output;
     }
+    
+    /**
+    * Helper displaying information message
+    * @param string $string|array
+    * @return string
+    */
+    public function displayInformation($string)
```
Comment (Quetzacoalt91) [inline classes/module/Module.php]: Be careful about the PHP doc and the variable name. `$string` is not anymore a proper name depending on its type.

```php
    /**
     * Helper displaying information message
     * @param string|array $information
     * @return string
     */
    public function displayInformation($information)
    {
        // [...]
```

Comment (SebBareyre) [reply classes/module/Module.php]: Yes but it is not the case in displayConfirmation

**Review by SebBareyre — COMMENTED** [review]
Voila again :)

File: classes/module/Module.php [hunk 2]
```diff
@@ -2050,24 +2050,24 @@ public function displayConfirmation($string)
     
     /**
     * Helper displaying information message(s)
-    * @param string $string|array
+    * @param string $information|array
```
Comment (PierreRambaud) [inline classes/module/Module.php]: should be `@param string|array $information`  :)

**Review by SebBareyre — COMMENTED** [review]
fix

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by marionf** [comment]
Hello @SebBareyre 
Can you fill the how to test section please ?
What type of information message ? On which pages ? When you do what ?
It will help to test your PR
Thanks :)

**Comment by SebBareyre** [comment]
@marionf Done.

**Comment by PierreRambaud** [comment]
Thanks @SebBareyre

**Pull request merged**